sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -373,9 +373,9 @@ AZ_NODISCARD az_result az_iot_hub_client_methods_response_get_publish_topic(
  */
 typedef enum
 {
-  AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_GET = 1,
-  AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES = 2,
-  AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES = 3,
+  AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_GET = 1,
+  AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES = 2,
+  AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES = 3,

```

Comment:
To be clear, looks like we are inserting the `_HUB_` as an in-fix as part of this rename. Everything else remains the same.